src/program.js

```diff
@@ -175,6 +177,22 @@ export class Program {
 // A global variable generated by DefinePlugin, generated in webpack.config.js
 declare var WEBEXT_BUILD_ENV: string;
 
+function setDefaultValues(options: Object): Object {
+  const defaultValues = {};
+  if (options !== undefined && options !== null) {
```

Comment:
why is this check necessary? The Flow definition of `options` says that it can't be undefined or null.